src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs

```diff
@@ -432,26 +435,47 @@ public Uri GetTeamProjectUrl(string projectName, string teamName)
             return builder.Uri;
         }
 
+
+        internal async Task HandleLoginAsync(CredentialPromptType showDialog=CredentialPromptType.DoNotPrompt, Uri serverUri = null)
```

Comment:
this might be easier to read if is refactored to:
```
serverUri = serverUri ?? Configuration.SavedConnection.ServerUri;
if (serverUri != null) {
   try {
      await...
   } catch () { … }
}
```

Reply:
Good idea. Also mixed in some of @RobGallo's nesting-avoidance approach